lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -40,11 +50,13 @@ class Classifier(MagmaController):
     """
     APP_NAME = "classifier"
     APP_TYPE = ControllerType.SPECIAL
+    SESSIOND_RPC_TIMEOUT = 5
+    CLASSIFIER_CONTROLLER_ID = 5
```

Comment:
why is this 5?

Reply:
For now, We defined the classifier controller id is constant value 5. 
We can move to as a configure parameter and will define in pipelined.yml file with value.

Reply:
sure lets do that

Reply:
Done.